services/web-server/src/utils/verifyJwt.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert');
+const jwt = require('jsonwebtoken');
+const jwks = require('jwks-rsa');
+
+module.exports = async ({ token, domain }) => new Promise((resolve, reject) => {
+  assert(token, 'No authorization token was found');
+  assert(domain, 'A domain is required to verify the jwt');
+
+  const client = jwks({
```

Comment:
I think we want to construct this once, not for every call -- it rate-limits itself and caches the value between calls.